release-testing/executors.sh

```diff
@@ -3,19 +3,31 @@
 . ./store.sh
 
 function exec_conjur_client() {
-  kubectl --namespace "$(store_get TEST_NAMESPACE)" \
-   exec -i "$(store_get CONJUR_CLIENT_POD_NAME)" \
+  local TEST_NAMESPACE CONJUR_CLIENT_POD_NAME
```

Comment:
I think all of the local variables in your bash scripts can be condensed from:
```
local NAME
NAME=foo
```
to
```
local NAME=foo
```

It should shorten the files by quite a bit and improve readability.

Reply:
For the literals yes, and I've tried to do that. For those that require variable substitution from the result of a sub-shell we have to split out the variable declaration and the variable assignment so that we can catch errors from the sub-shells. Inspiration taken from shellcheck